custom_components/easee/easee.py

```diff
@@ -46,7 +46,74 @@ async def get_consumption_between_dates(self, from_date, to_date):
         return float(value)
 
     async def start(self):
-        return await self.session.post(f"/api/chargers/{self.id}/commands/start_charging")
+        """Start charging session"""
+        return await self.easee.post(f"/api/chargers/{self.id}/commands/start_charging")
+
+    async def pause(self):
+        """Pause charging session"""
+        return await self.easee.post(f"/api/chargers/{self.id}/commands/pause_charging")
+
+    async def resume(self):
+        """Resume charging session"""
+        return await self.easee.post(
+            f"/api/chargers/{self.id}/commands/resume_charging"
+        )
+
+    async def stop(self):
+        """Stop charging session"""
+        return await self.easee.post(f"/api/chargers/{self.id}/commands/stop_charging")
+
+    async def toggle(self):
+        """Toggle charging session start/stop/pause/resume """
+        return await self.easee.post(
+            f"/api/chargers/{self.id}/commands/toggle_charging"
+        )
+
+    async def get_basic_charge_plan(self):
+        """Get and return charger basic charge plan setting from cloud """
+        return await self.easee.get(
+            f"/api/chargers/{self.id}/commands/basic_charge_plan"
+        )
+
+    async def set_basic_charge_plan(
+        self, id, chargeStartTime, chargeStopTime, repeat=True
+    ):
+        """Set and post charger basic charge plan setting to cloud """
+        json = {
+            "id": id,
+            "chargeStartTime": chargeStartTime,
+            "chargeStopTime": chargeStopTime,
+            "repeat": repeat,
+        }
+        return await self.easee.post(
+            f"/api/chargers/{self.id}/commands/basic_charge_plan", json=json
+        )
+
+    async def delete_basic_charge_plan(self):
+        """Delete charger basic charge plan setting from cloud """
+        return await self.easee.post(
+            f"/api/chargers/{self.id}/commands/basic_charge_plan"
```

Comment:
Can you only have one plan?

Reply:
Seems like it. I had the same question when testing, but I see in my app (android) that there is also just one plan. Perhaps this is something that is not 100% completed in the API (see comment about id).